featuretools/tests/primitive_tests/test_transform_primitive.py

```diff
@@ -174,6 +176,93 @@ def test_is_quarter_start():
     np.testing.assert_array_equal(iqs_bools, correct_bools)
 
 
+def test_is_lunch_time():
+    ilt = IsLunchTime()
+    dates = pd.Series(
+        [
+            datetime(2022, 6, 26, 12, 12, 12),
+            datetime(2022, 6, 21, 12, 3, 4),
+            datetime(2022, 6, 21, 11, 3, 4),
+            np.nan,
+        ]
+    )
+    actual = ilt(dates)
+    expected = [True, True, False, False]
+    np.testing.assert_array_equal(actual, expected)
+
+
+def test_is_lunch_time_weekdays_only():
+    ilt = IsLunchTime(include_weekends=False)
+    dates = pd.Series(
+        [
+            datetime(2022, 6, 26, 12, 12, 12),
+            datetime(2022, 6, 21, 12, 3, 4),
+            datetime(2022, 6, 21, 11, 3, 4),
+            datetime(2022, 7, 4, 12, 1, 1),
+        ]
+    )
+    actual = ilt(dates)
+    expected = [False, True, False, False]
+    np.testing.assert_array_equal(actual, expected)
+
+
+def test_is_lunch_time_include_holidays():
+    ilt = IsLunchTime(include_weekends=False, include_holidays=True)
+    dates = pd.Series(
+        [
+            datetime(2022, 6, 26, 12, 12, 12),
```

Comment:
can you include a comment that is something like `Wed, Jun 26 2022 - (Holiday Name)`